packages/patternfly-4/react-core/src/beta/components/Drawer/examples/Drawer.md

```diff
@@ -49,7 +49,7 @@ class SimpleDrawer extends React.Component {
 
 ```js title=Basic-inline
 import React, { ReactFragment } from 'react';
-import { Drawer, DrawerPanelContent, DrawerContent } from '@patternfly/react-core/dist/esm/experimental';
+import { Drawer, DrawerPanelContent, DrawerContent } from '@patternfly/react-core/dist/esm/beta';
```

Comment:
Can these be changed to @patternfly/react-core directly now?

Reply:
I agree for new components. I actually think we may want to still export from experimental (and from react-core) for existing components and remove that with the breaking change release.

Reply:
I agree - for this, I'm just talking about the import statements we are using - I'd rather that we just treat these components like other components to indicate the commitment to them.    That does remind me that I need to look to see if the experimental/index.js still exists...

Reply:
Ok think I see what you mean, pushed some changes - let me know if I missed anything!